src/server/services/generate-arpa-report.js

```diff
@@ -199,7 +199,7 @@ async function generateProject19 (records) {
             record.content.Tertiary_Proj_Demographics_Explanation__c,
             record.content.Structure_Objectives_of_Asst_Programs__c,
             record.content.Recipient_Approach_Description__c,
-            record.content.Number_Non_Profits_Served__c
+            record.content.Small_Businesses_Served__c ?? record.content.Number_Non_Profits_Served__c
```

Comment:
EC 1.9 I think actually DOES want the number of Non-profits, and this line should be reverted 
![Screen Shot 2022-07-22 at 8 55 55 AM](https://user-images.githubusercontent.com/3675290/180477744-f4e44240-fb0a-4778-a126-4f1117164853.png)
d

Reply:
Thanks, you're definitely right!

In fact, it looks like I specifically changed this field in 1.8 to the wrong value when making the last round of updates.  I guess that measn we shouldn't need to keep the fallback lookup in template 1.8 either.

I wonder how I made that mistake.